### PR TITLE
any2fasta 0.4.2 (new formula)

### PR DESCRIPTION
**Pull request by tseemann**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by sjackman** [comment]
I prefer to put short build scripts directly in the `meta.yml`. What's your preference?

File: recipes/any2fasta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```
[the remark this answers is not in the document]
Comment (tseemann) [reply recipes/any2fasta/meta.yaml]: Arghhh.  i thought you counted from 1. Sorry!

**Comment by tseemann** [comment]
@sjackman i didn't even know you could put build instructions in `meta.yml`.

Next time i will not accept my own pull request. It was bad form.

**Comment by sjackman** [comment]
No worries, Torsten! Here's an example of an inline script:
https://github.com/bioconda/bioconda-recipes/blob/0c15cb167ffc448e20525bca0b28e9835e4c4a2c/recipes/abeona/meta.yaml#L17
You can also do
```yaml
script: |
  ./configure
  make
```

**Pull request merged**